tcl/board.tcl

```diff
@@ -1422,7 +1423,7 @@ proc ::board::update {w {board ""} {animate 0}} {
     $w.bd delete p$sq
     $w.bd create image $xc $yc -image $::board::letterToPiece($piece)$psize -tag p$sq
   }
-  ::board::innercoords $w
+  $w.bd raise coords
```

Comment:
The tag should be called "innercoords" instead of "coords"
Invoking raise is not expensive but it is not free:
https://github.com/tcltk/tk/blob/main/generic/tkCanvas.c#L1782
so we should call it only if there are innercoords:
```
if {$::board::_coords($w) >= 3 } {
  $w.bd raise innercoords
```